jax/_src/api.py

```diff
@@ -2611,6 +2611,7 @@ def __init__(self, shape, dtype):
 
 
 def checkpoint(fun: Callable, concrete: bool = False, prevent_cse: bool = True,
+               policy: Optional[Callable[..., bool]] = None,
```

Comment:
Wasn't the plan to expose a few pre-packaged policies for now? Exposing primitives and their names might make future internal changes very breaking